pkg/watermark/publish/publisher.go

```diff
@@ -48,22 +48,25 @@ type Publisher interface {
 	GetLatestWatermark() wmb.Watermark
 }
 
+var initialWatermark = wmb.Watermark(time.UnixMilli(-1))
```

Comment:
```suggestion
const InitialWatermark = wmb.Watermark(time.UnixMilli(-1))
```

Can we also move this constant to pkg/watermark/wmb/wmb.go? I think it can be used across files under watermark.